tests/framework/Samplers/AdaptiveMonteCarlo/Adapt/adaptive_test_model.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import math
+def run(self,Input):
```

Comment:
add a docstring for this to explain the type of model you are using (I know it is quite clear but repetita iuvant)